web/components/inputs/FcInputProjectSearch.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="fc-input-project-search">
+    <v-autocomplete
```

Comment:
this makes me wonder if a `<v-autocomplete>`-based implementation of `FcInputLocationSearch` would be possible.  (if so, it would _definitely_ reduce complexity there!)